dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/FormLoginTest.java

```diff
@@ -51,39 +51,48 @@
 
 import static org.junit.Assert.assertNotNull;
 
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import com.ibm.ws.webcontainer.security.test.servlets.FormLoginClient;
 import com.ibm.ws.webcontainer.security.test.servlets.FormLoginClient.LogoutOption;
 import com.ibm.ws.webcontainer.security.test.servlets.SSLFormLoginClient;
 
+import componenttest.annotation.CheckpointTest;
 import componenttest.custom.junit.runner.FATRunner;
-import componenttest.custom.junit.runner.Mode;
-import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.CheckpointRule;
+import componenttest.rules.repeater.CheckpointRule.ServerMode;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.impl.LibertyServerFactory;
 import componenttest.topology.utils.LDAPUtils;
 
 @RunWith(FATRunner.class)
-@Mode(TestMode.FULL)
+@CheckpointTest(alwaysRun = true)
 public class FormLoginTest extends CommonServletTestScenarios {
     private static LibertyServer myServer = LibertyServerFactory.getLibertyServer("com.ibm.ws.webcontainer.security.fat.formlogin");
     private static Class<?> myLogClass = FormLoginTest.class;
     private static FormLoginClient myClient;
     private static SSLFormLoginClient mySSLClient;
 
-    @BeforeClass
-    public static void setUp() throws Exception {
+    @ClassRule
+    public static CheckpointRule checkpointRule = new CheckpointRule().setConsoleLogName(FormLoginTest.class.getSimpleName()
+                                                                                         + ".log").setServerSetup(FormLoginTest::serverSetUp).setServerStart(FormLoginTest::serverStart).setServerTearDown(FormLoginTest::serverTearDown);
```

Comment:
This would be more readable if you forced some line breaks like this.
```suggestion
    public static CheckpointRule checkpointRule = new CheckpointRule() //
                    .setConsoleLogName(FormLoginTest.class.getSimpleName() + ".log") //
                    .setServerSetup(FormLoginTest::serverSetUp) //
                    .setServerStart(FormLoginTest::serverStart) //
                    .setServerTearDown(FormLoginTest::serverTearDown);
```

Reply:
Done, thanks!